BookingsApi/BookingsApi.DAL/Migrations/BookingsDbContextModelSnapshot.cs

```diff
@@ -171,6 +171,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<int>("CaseTypeId")
                         .HasColumnType("int");
 
+                    b.Property<int?>("ConferenceSupplier")
```

Comment:
I don't think this should be nullable. A booking/conference must have a supplier. In video API I'm defaulting it to Kinly, can we do the same here?

then if necessary, we can run a script to update existing bookings to vodafone if necessary

Reply:
just wondering, do we even need to store the supplier in bookings API? Isn't it just going to be passed straight to BQS and in turn video API?

Reply:
it's got to be nullable because there are existing bookings, some of which are already a mix of kinly and vodafone. If I default to Kinly, it will probably tank any existing bookings.
You're right maybe this doesn't need to be saved in Bookings API since this is more of a Video API concern. Let's have a call about this since I'm making admin web changes now too

Reply:
After a discussion, @oliver-scott and I have agreed we do need to keep this in bookings api because hearings that are booked without a judge do not get published to the service bus so the original value needs to be tracked somewhere